TelepresenceBot/mobile/src/main/java/com/novoda/tpbot/bot/device/usb/UsbChangesRegister.java

```diff
@@ -0,0 +1,57 @@
+package com.novoda.tpbot.bot.device.usb;
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.content.IntentFilter;
+import android.hardware.usb.UsbManager;
+import android.support.annotation.Nullable;
+
+class UsbChangesRegister {
+
+    static final String ACTION_USB_PERMISSION = "com.novoda.tpbot.USB_PERMISSION";
+
+    private final Context context;
+
+    @Nullable
+    private UsbChangesBroadcastReceiver usbChangesBroadcastReceiver;
+
+    UsbChangesRegister(Context context) {
+        this.context = context;
+    }
+
+    void register(UsbChangesListener usbChangesListener) {
+        if (usbChangesBroadcastReceiver == null) {
```

Comment:
Only register one broadcast receiver at a time. Similar to what we do for binding services.